crates/control/src/behavior/node.rs

```diff
@@ -38,11 +40,11 @@ pub struct CreationContext {
 pub struct CycleContext {
     pub path_obstacles: AdditionalOutput<Vec<PathObstacle>, "path_obstacles">,
     pub active_action: AdditionalOutput<Action, "active_action">,
+    pub time_to_reach_obstacles: AdditionalOutput<Vec<PathObstacle>, "time_to_reach_obstacles">,
```

Comment:
What is the point of this additional output?

Reply:
This was moved out of `dribble_path_planner` since it is not a node anymore. @MaikRe why did you need this?

Reply:
More specifically, the `path_obstacles` additional output is currently empty when the robot dribbles.
The dribble action used to fill this output but now that the dribble path is precalculated, it only fills the `time_to_reach_obstacles` (which might not be the best name anyways).
Are there cases where we'd want to have the dribble path obstacles separately even though the robot does something else?
If not, I'd say we should merge the two, otherwise, the dribble obstacles should be copied to the path obstacles when the robot is executing the dribble action.

Reply:
Good point. I renamed the `time_to_reach_obstacles` to `dribble_path_obstacles` and filled the `path_obstacles` accordingly.